lib/sdf-server/src/service/session.rs

```diff
@@ -93,6 +93,8 @@ pub fn routes() -> Router<AppState> {
         .route("/load_workspaces", get(load_workspaces::load_workspaces))
         .route(
             "/refresh_workspace_members",
-            post(refresh_workspace_members::refresh_workspace_members),
+            post(refresh_workspace_members::refresh_workspace_members).layer(
```

Comment:
Just for my own understanding - anyone who is an owner of a change set, whenever they change someone's permissions, this will kick in and here we're ensuring that only owners can refresh the workspace members, yeah?

Reply:
Ah yes, that's basically it!

Reply:
that is already how the Auth API does it :)